test/WebJobs.Script.Tests/Binding/ExtensionBindingTests.cs

```diff
@@ -1,72 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-#if ALLEXTENSIONS
-using System.Linq;
-using Microsoft.Azure.WebJobs.Script.Binding;
-using Xunit;
-
-namespace Microsoft.Azure.WebJobs.Script.Tests
-{
-    public class ExtensionBindingTests
-    {
-        [Fact]
-        public void GetAttributeBuilderInfo_ServiceBusTriggerAttribute_Topic()
-        {
-            ServiceBusTriggerAttribute attribute = new ServiceBusTriggerAttribute("myTopic", "mySubscription", Microsoft.ServiceBus.Messaging.AccessRights.Listen);
-            var builderInfo = ExtensionBinding.GetAttributeBuilderInfo(attribute);
```

Comment:
These are the only unit tests for the GetAttributeBuilderInfo method. We want to strive have unit tests for all non-trivial functions (like this one). In this case, I used these tests to develop/debug this method.

If your goal is to remove SB specifics, you could create a test attribute with the same/similar structure and have these tests target that. We need to keep the tests though.

Reply:
I"ll create a test attribute.